src/main/kotlin/snyk/common/SnykCachedResults.kt

```diff
@@ -38,68 +49,126 @@ class SnykCachedResults(val project: Project) {
         currentOssError = null
         currentContainerError = null
         currentIacError = null
+//        currentSnykCodeResultsLS = null
```

Comment:
nit: commented out code left behind

Reply:
Or actually, do we not want to clean these results too? I can see `cleanCaches` is used when triggering a rescan for example

Reply:
https://github.com/snyk/snyk-intellij-plugin/pull/480